config/src/config/secure_backend_config.rs

```diff
@@ -164,6 +168,43 @@ impl OnDiskStorageConfig {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct RocksDbStorageConfig {
+    // Required path for on disk storage
+    pub path: PathBuf,
+    /// A namespace is an optional portion of the path to a key stored within OnDiskStorage. For
+    /// example, a key, S, without a namespace would be available in S, with a namespace, N, it
+    /// would be in N/S.
+    pub namespace: Option<String>,
+    #[serde(skip)]
+    data_dir: PathBuf,
+}
+
+impl Default for RocksDbStorageConfig {
+    fn default() -> Self {
+        Self {
+            namespace: None,
+            path: PathBuf::from(SECURE_STORAGE_DB_NAME),
+            data_dir: PathBuf::from("/opt/aptos/data"),
```

Comment:
We should really put these default paths all in one place

Reply:
Agreed, I've just copy/pasted the `OnDiskStorageConfig`. We should follow up and clean these things up post cut.